tests/app/count.js

```diff
@@ -30,27 +30,32 @@ define([
       console.log = function (val) {
         nums.push(val);
       };
+
+      this.clock = sinon.useFakeTimers();
     });
 
     afterEach(function () {
       console.log = origConsoleLog;
+
+      this.clock.restore();
     });
 
     it('should count from start number to end number, one per 1/10th of a second', function (done) {
       this.timeout(600);
       answers.count(1, 5);
 
-      setTimeout(function () {
-        expect(nums.length > 1).to.be.ok;
-        expect(nums.length < 5).to.be.ok;
-      }, 200);
-
-      setTimeout(function () {
-        expect(nums.length).to.eql(5);
-        expect(nums[0]).to.eql(1);
-        expect(nums[4]).to.eql(5);
-        done();
-      }, 550);
+      this.clock.tick(200);
+
+      expect(nums.length > 1).to.be.ok;
+      expect(nums.length < 5).to.be.ok;
+
+      this.clock.tick(350);
+
+      expect(nums.length).to.eql(5);
+      expect(nums[0]).to.eql(1);
+      expect(nums[4]).to.eql(5);
```

Comment:
We can be even more precise here, I think, by literally ticking the clock 1/10th of a second and verifying that it's counting by one with each tick. It'll be a bit more code, and will be kinda repetitious, but it'll make sure that the solution that is input is actually right.

It might also be overkill tho'. Thoughtz?